package.json

```diff
@@ -57,10 +56,10 @@
     "eslint-plugin-react": "^7.32.2",
     "eslint-plugin-react-hooks": "^4.6.0",
     "jest": "^29.2.1",
-    "jest-expo": "^48.0.2",
+    "jest-expo": "~51.0.2",
     "patch-package": "^6.5.1",
     "prettier": "^2.8.6",
-    "ts-node": "^10.9.1",
-    "typescript": "^4.9.4"
+    "ts-node": "^10.9.2",
```

Comment:
Fixes Jest not launching on latest TS version: https://github.com/TypeStrong/ts-node/issues/2076#issuecomment-2074645988